python/nav/web/templates/navlets/room_location_status_edit.html

```diff
@@ -1,10 +1,9 @@
 {% extends 'navlets/base.html' %}
-{% load crispy_forms_tags %}
 
 {% block navlet-content %}
 
   <form action="{% url 'get-user-navlet' navlet.navlet_id %}">
-    {% crispy form %}
+    {% include 'custom_crispy_templates/flat_form.html' %}
```

Comment:
Shouldnt it be `{% include 'custom_crispy_templates/_form_content.html' %}` since its included inside a form? Original form also says `self.helper.form_tag = False`

Reply:
True, good that you caught that!